src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -1509,16 +1509,20 @@ public override BoundNode VisitUsingStatement(BoundUsingStatement node)
 
             foreach (LocalSymbol local in localsOpt)
             {
-                if (local.DeclarationKind == LocalDeclarationKind.RegularVariable)
-                {
-                    ReportIfUnused(local, assigned: true);
-                }
-                else
+                switch (local.DeclarationKind)
                 {
-                    NoteRead(local); // At the end of the statement, there's an implied read when the local is disposed
+                    case LocalDeclarationKind.RegularVariable:
```

Comment:
Never mind, I know why: sometimes the syntax requires you to declare a variable you have no intention to use.